crates/viewer/re_viewer/src/app.rs

```diff
@@ -1836,7 +1836,10 @@ impl eframe::App for App {
         }
 
         // Temporarily take the `StoreHub` out of the Viewer so it doesn't interfere with mutability
-        let mut store_hub = self.store_hub.take().unwrap();
+        let mut store_hub = self
+            .store_hub
+            .take()
+            .expect("Failed to take store hub");
```

Comment:
D'oh `cargo fmt` flattens this now that I shortened the expect message, which is not good consistency with the other updates I made and not what I really wanted to happen here - argh